apps/slackbot/user/user_usage_tracker.py

```diff
@@ -0,0 +1,175 @@
+import sqlite3
+import time
+
+
+class UserUsageTracker:
+    def __init__(self, db_name='token_countersaa.db', current_time=int(time.time()), max_daily_token=20000):
+        self.conn = sqlite3.connect(db_name)
```

Comment:
So, we are using a file-based DB here, which is good for local development. But will the file still persist in deployment if we deploy a new version?

Reply:
no . but until we have a need for another relational database using this. I believe it is efficient enough but if this is something crucial. let's create another PR for it because the limiter can work until then.

Reply:
I think this looks good for now. We can add the relational database in the future.